### PR TITLE
Add Default CCIs for Relevant NIST Controls

### PR DESCRIPTION
**Pull request by camdenmoors**

### CODE REVIEW

**Review by em-c-rod — COMMENTED** [review]
From a high level, it looks like these changes are generally to support a new tag field for CCIs. Is that option new? Or just wasn't in the testing before? Anything else to pay close attention to?

**Comment by camdenmoors** [comment]
> From a high level, it looks like these changes are generally to support a new tag field for CCIs. Is that option new? Or just wasn't in the testing before? 
This is a new feature to support conversion to POA&M

> Anything else to pay close attention to?
That the data should be included on import and matches that of the included lookup table

File: libs/hdf-converters/src/twistlock-mapper.ts [hunk 1]
```diff
@@ -67,7 +68,7 @@ export class TwistlockMapper extends BaseConverter {
             key: 'id',
             tags: {
               nist: ['SI-2', 'RA-5'],
-              cci: ['CCI-002605', 'CCI-001643'],
+              cci: getCCIsForNISTTags(['SI-2', 'RA-5']),
```
Comment (Amndeep7) [inline libs/hdf-converters/src/twistlock-mapper.ts:71]: Don't we have a constant somewhere that defines our default nist tags?  Use that.

File: libs/hdf-converters/src/utils/global.ts [hunk 2]
```diff
@@ -50,3 +54,16 @@ export function getDescription(
 
   return found;
 }
+
+export function getCCIsForNISTTags(nistTags: string[]): string[] {
+  const cciTags: string[] = [];
+  nistTags.forEach((tag) => {
+    const baseTag = /\w\w-\d/g.exec(tag);
+    if (baseTag) {
+      if (baseTag[0] in NistCciMappingData) {
```
Comment (Amndeep7) [inline libs/hdf-converters/src/utils/global.ts:63]: combine the two conditionals with && in order to get rid of pointless nesting

**Review by Amndeep7 — CHANGES_REQUESTED** [review]

**Comment by Amndeep7** [comment]
@camdenmoors bump

**Comment by Amndeep7** [comment]
@camdenmoors can you fix the merge conflict?  Also the veracode mapper was merged, so can you add the cci stuff there as well?

**Pull request merged**